Source/Constraint.swift

```diff
@@ -61,6 +61,11 @@ public class Constraint {
     internal var makerFile: String = "Unknown"
     internal var makerLine: UInt = 0
     
+    public var layoutConstraint: LayoutConstraint? {
```

Comment:
Shame on me! I did expose this on my fork to allow some constraint animations with Pop, I should have excluded this from the PR >_<'